src/frontend/screens/Game/GamePage/components/HLTBInline.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useContext, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import GameContext from '../../GameContext'
+import { Speed, ExpandMore } from '@mui/icons-material'
+import { Accordion, AccordionSummary, AccordionDetails } from '@mui/material'
+import HowLongToBeat from 'frontend/components/UI/WikiGameInfo/components/HowLongToBeat'
+
+const HLTBInline = () => {
+  const { t } = useTranslation('gamepage')
+  const { wikiInfo } = useContext(GameContext)
+
+  const [isExpanded, setIsExpanded] = useState(false)
+
+  function handleChange() {
+    setIsExpanded((prevExpanded) => !prevExpanded)
+  }
+
+  if (!wikiInfo) {
+    return null
+  }
+
+  const howlongtobeat = wikiInfo.howlongtobeat
+
+  if (!howlongtobeat) {
+    return null
+  }
+
+  return (
+    <div className="hltbWrapper">
+      <Accordion expanded={isExpanded} onChange={handleChange}>
+        <AccordionSummary
+          expandIcon={<ExpandMore />}
+          aria-control="hltb-content"
```

Comment:
I get an invalid prop warning in the console with this

![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/188464/dea5b9bf-ba24-4831-95de-89a9e8ab5f09)